packages/training/deploy/runpod/setup.py

```
@@ -0,0 +1,251 @@
+                if value:  # Only set non-empty values
+                    env[key] = value
+    return env
+
+
+def api(method, endpoint, data=None):
+    """Make API request."""
+    key = os.environ.get("RUNPOD_API_KEY")
+    if not key:
+        sys.exit("Set RUNPOD_API_KEY (https://console.runpod.io/user/settings)")
+    
+    r = requests.request(
```

Ruff (0.14.13)

Probable use of `requests` call without timeout

(S113)